planningcenter/grids/horizontal-main-nav/stories/3-horizontal-main-nav--components.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import { Meta, Story, Canvas, Props } from "@storybook/addon-docs/blocks";
+
+import HorizontalMainNav from "../horizontal-main-nav.tsx";
+import "../horizontal-main-nav.css";
+
+import { SampleContainer } from "./helpers.tsx";
+
+<Meta title="Grids/Horizontal Main Nav/Components" />
+
+# Horizontal Main Nav: Components
+
+## React
+
+There is one component exported for implementation convenince.
+
+Sample
+
+<Canvas>
+  <SampleContainer>
+    <HorizontalMainNav
+      id="a-discrete-id"
+      label="Sample resource"
+      navItems={[
+        <a key="current" href="#" aria-current={true}>
+          Current
+        </a>,
+        <a key="occluded" href="#">
+          Occluded
+        </a>,
+      ]}
+      actionItems={[<button>Action 1</button>, <button>Action 2</button>]}
+    />
+  </SampleContainer>
+</Canvas>
+
+### Interface
+
+<Props of={HorizontalMainNav} />
+
+### Implementation
+
+A default composition of Horizontal Main Nav is provided as a convenience for the majority of cases.
+
+This sample shows a complete use of the available control props.
+
+Note:
+
+- `label` is used to construct the read `aria-label` of the nested `<nav>` elemnt
```

Comment:
I love that accessibility is considered and mentioned here!! FYI there's a typo on the last word here. "elemnt"